data/resources/style.css

```diff
@@ -148,6 +148,36 @@ sidebarsearchitemrow {
   color: @accent_fg_color;
 }
 
+messagedocument > label.sender-text {
+  margin-bottom: 3px;
+}
+
+messagedocument > box {
+  padding: 9px;
+  transition: 200ms;
+}
+
+messagedocument > box > image {
+  background: @accent_color;
+  color: @window_bg_color;
+  min-width: 40px;
+  min-height: 40px;
+  border-radius: 9999px;
+}
+
+messagedocument.outgoing > box > image {
+  background: @accent_fg_color;
+  color: @accent_bg_color;
+}
+
+messagedocument > box:hover {
+  background: alpha(currentColor, 0.1);
+}
+
+messagedocument > box:active {
+  background: alpha(currentColor, 0.2);
+}
```

Comment:
But all message is clickable area

Reply:
> But all message (or document) is clickable area

I know, but visually it still looks weird to highlight the entire bubble, so in this case it will only highlight the "button".